src/Taboo/TabooTeam.py

```diff
@@ -34,3 +34,7 @@ def getPlayer(self, playerName):
             Return None if no such player found
         """
         return self.members.get(playerName, None)
+
+    def ready(self):
+        return (len(self.members) > 0 and
+                    all(plyr.ready for plyr in self.members.values()))
```

Comment:
this should be >= 2 to ensure the game doesn't start with just 1 player otherwise the team might get "stuck" in the middle of the game?

I wonder if we want to do this the long way (this allows adding trace messages):

   if len(self.members) < 2:
      trace(Level.info, "Team", self.teamNumber, "has", len(self.members), " (less than two) players")
      return False

   if all(plyr.ready ...):
      return True

   trace(Level.info, "Team", self.teamNumber, "some players are not ready")
   return False

Reply:
Agree. Will be fixed in next pull request